### ISSUE TITLE
Chosen dropdown navigation issue with JAWS tool.

### ISSUE THREAD

**Issue by ArunM2017** [harvesthq/chosen#2874]
Hi,

We are using 'Chosen' drop down in our project.

In JAWS tool testing we found one issue related to reading of drop down items.

[https://thepaciellogroup.github.io/AT-browser-tests/](https://thepaciellogroup.github.io/AT-browser-tests/)

Above hyperlink describes “General Navigation Notes” available associated with JAWS testing.
According to this link user need to use the key “I” to read the next item within drop down and use “SHIFT + I” for reading previous item within drop down.

This is working fine in Chrome Browser. But Not working in IE browser.

This scenario will reproduce in the following link drop down which is under the label “Single Select”.(Please refer attached screenshot). This drop down is similar to the one which we are using in our project.

[https://harvesthq.github.io/chosen/](https://harvesthq.github.io/chosen/)

![chosen dropdown](https://user-images.githubusercontent.com/31404965/29770072-d030db8a-8c0a-11e7-8215-3a709279ee77.png)

Version details are as below.

Chrome Browser Version : 60.0.3112.101 (Official Build) (64-bit)

IE browser Version : 11.1045.10586.0CO

OS : Windows 10

JAWS tool version : 18.0

Please update any solution available for this issue.

**Comment by koenpunt** [comment]
If I understand correctly; this is about accessibility?

**Comment by ArunM2017** [comment]
Yes it is accessibility issue. JAWS tool is not reading the 'Chosen' drop down items in IE browser.

**Comment by ArunM2017** [comment]
Hi,

Any update on this issue ? Please update anyone.

**Comment by mgifford** [comment]
This might be a JAWS or IE issue. Does it work in the latest Edge? IE support is dropping like flies.
